pages/literature.md

```diff
@@ -22,7 +22,7 @@ layout: default
 
 ### [13th International Modelica Conference 2019](https://2019.international.conference.modelica.org/)
 
-FMI User meeting ([Agenda](https://www.modelica.org/events/modelica2019/proceedings/html/userpres.html)):
+FMI User meeting ([Agenda](https://2019.international.conference.modelica.org/proceedings/html/userpres.html)):
 
 * [Overview of the FMI, DCP and SSP standards](https://svn.fmi-standard.org/fmi/branches/public/docs/Modelica2019/Industrial_User_Presentations_Modelica2019/001_Overview_FMI_SSP_DCP.pdf)
```

Comment:
all https://svn.fmi-standard.org links are broken

Reply:
I think this should be done later - I am running into so many merge conflicts and have issues with tooling here, I would like to get this improvement in use and later fix more...